app/inputs/nested_select_input.rb

```diff
@@ -94,11 +94,11 @@ def select_classes(level_data)
   end
 
   def model_name
-    @object.class.to_s.downcase
+    @object.class.to_s.underscore.gsub('/', '_')
```

Comment:
If `@object.class` is ,for example, `"SectionWorker"` I think is nicer to have:
`@object.class.to_s.underscore` = `"section_user"` instead of:
`@object.class.to_s.downcase` = `"sectionuser"`
but this is not an error.
You have added this part: `underscore.gsub('/', '_')`. Why did you do this?
do you have your `ActiveRecord` models inside a namespace?